tf/main.tf

```diff
@@ -0,0 +1,42 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+module "lambda_function" {
+  source = "terraform-aws-modules/lambda/aws"
+
+  function_name = "smoke-testing-test"
+  description   = "Example Lambda function for smoke testing"
+  handler       = "index.handler"
+  runtime       = "nodejs16.x"
+
+  source_path = "../testdata/function/"
+
+  layers = [
+    module.lambda_layer_local.lambda_layer_arn,
+    "arn:aws:lambda:eu-central-1:267093732750:layer:elastic-apm-node-ver-3-38-0:1",
+  ]
+
+  environment_variables = {
+    NODE_OPTIONS                  = "-r elastic-apm-node/start"
+    ELASTIC_APM_LOG_LEVEL         = var.log_level
+    ELASTIC_APM_LAMBDA_APM_SERVER = "CHANGEME"
+    ELASTIC_APM_SECRET_TOKEN      = "CHANGEME"
```

Comment:
Should these be variables?

Reply:
Updated to use the apm server and secret token from the ephemeral deployment